.spelling

```diff
@@ -71,7 +71,6 @@ storybook.js.org
 YuzuJS
 setImmediate
 Malte
-Ubl
```

Comment:
I'm assuming that this is similar to a file I have, I can find no particular reason for this item to exist.